README.md

```diff
@@ -9,85 +9,117 @@ website: cltl.nl
 https://github.com/cltl/StoryTeller
 
 ## DESCRIPTION of the StoryTeller stack
-The Storyteller system will allow you to generate general queries about topics on the [KnowledgeStore](https://knowledgestore.fbk.eu/) and visualize the results for analysis and exploration. It tries to accomplish this task by providing the user with web frontends that simplify the information needed for the complex tasks required and allowing the user to explore the knowledge with custom selections.
+
+The Storyteller system will allow you to generate general queries about topics
+on the [KnowledgeStore](https://knowledgestore.fbk.eu/) and visualize the
+results for analysis and exploration. It tries to accomplish this task by
+providing the user with web frontends that simplify the information needed for
+the complex tasks required and allowing the user to explore the knowledge with
+custom selections.
 
 ## Installation
 ### Automated installation of the complete query building and visualization suite
+
 (please make sure to install docker and docker-compose https://www.docker.com/)  
 
-After installation of docker, the following bash commands should suffice to install the entire system:
+After installation of docker, the following bash commands should suffice to
+install the entire system:
 
 ```bash
-    docker volume create --name=data
-    docker-compose up
+docker volume create --name=data
+docker-compose up
 ```
-This will create a set of docker containers that will run the following system for you:  
-![diagram of the stack](https://cdn.rawgit.com/NLeSC-Storyteller/StoryTeller/master/doc/stack-diagram.svg "Stack diagram")  
+This will create a set of docker containers that will run the following system
+for you:   ![diagram of the
+stack](https://cdn.rawgit.com/NLeSC-Storyteller/StoryTeller/master/doc/stack-diagram.svg
+"Stack diagram")  
 
-Please allow the installation process some time. After installation is complete, the web frontends can be found at [localhost:9000](http://localhost:9000) for the query builder component, and at [localhost:9001](http://localhost:9001) for the visualization component
+Please allow the installation process some time. After installation is complete,
+the web frontends can be found at [localhost:9000](http://localhost:9000) for
+the query builder component, and at [localhost:9001](http://localhost:9001) for
+the visualization component
 
 ### Automated installation of this component only
-*Please Note that this will also install all necessary dependencies in the docker container*  
+
```

Comment:
Not sure why the information from this section needs to be here.

Also, what does 'this component' refer to? NLeSC-StoryTeller/StoryTeller?